sample/src/main/kotlin/com.github.drjacky.imagepicker/sample/MainActivity.kt

```diff
@@ -140,7 +140,7 @@ class MainActivity : AppCompatActivity() {
     fun pickCameraImage(view: View) {
         cameraLauncher.launch(
             ImagePicker.with(this)
-                .crop()
+                //.crop()
```

Comment:
Why's that?

Reply:
Okay, I see that's because of https://github.com/Drjacky/ImagePicker/issues/96

Reply:
but what if someone wants to crop the camera image?

Reply:
It's optional. Both with or without crop are handled.

Reply:
There is no problem. It's optional. Both with or without crop are handled.

Reply:
So, the issue is not with the crop itself as the titles say: `#96 Fix picking camera image without cropping` & `[Crash] When picking a camera image without setting crop()`

The improvement is here, right?
```kotlin
mImageUri?.path?.let { path ->
                File(path).delete()
            }
            it.delete() //Why uncommented BTW?
```